conans/client/generators/markdown.py

```diff
@@ -152,9 +152,9 @@
     $ cmake . -DCMAKE_TOOLCHAIN_FILE=cmake-build-release/conan_toolchain.cmake
     $ cmake --build .
 
-    # for Windows and Visual Studio 2017
+    # for Windows and Visual Studio 2022
```

Comment:
As VS 2022 is not still active in ConanCenter, there are no binaries for it, not sure if we should keep a previous VS version?

Reply:
I see. I have reverted this change.